rest_framework/fields.py

```diff
@@ -59,7 +59,11 @@ def is_simple_callable(obj):
     if not (function or method):
         return False
 
-    args, _, _, defaults = inspect.getargspec(obj)
+    if six.PY2:
+        args, _, _, defaults = inspect.getargspec(obj)
+    else:
+        args, _, _, defaults = inspect.getfullargspec(obj)[:4]
+
```

Comment:
See [here](https://github.com/rpkilby/django-rest-framework/blob/3bdb0e9dd8b8f098111fad4a874ee99d392e944a/rest_framework/fields.py#L52-L77) in #4505.